src/java.base/share/classes/java/util/Locale.java

```diff
@@ -455,10 +455,10 @@
  *
  * <p>For the backward compatible behavior, the system property
  * {@systemProperty java.locale.useOldISOCodes} reverts the behavior
- * back to prior to Java SE 17 one. If the system property is set
- * to {@code true}, those three current language codes are mapped to their
- * backward compatible forms. It is only read at the Java runtime startup,
- * so the later call to {@code System.setProperty()} won't affect the backward
+ * back to that of before Java SE 17. If the system property is set to
+ * {@code true}, those three current language codes are mapped to their
+ * backward compatible forms. It is only read at Java runtime startup, so a
```

Comment:
I had thought about some of some minor word smithing in your prior commit, but chose not to request a change.

In the above sentence,  It could be clearer what "It" is.  Perhaps something along the lines of:

This property is only read at Java runtime startup  and subsequents calls to ...."

Reply:
Also, "backward compatible behavior" maybe not needed to be repeated, e.g. "subsequent calls to ... will have no effect.

"back to that of before Java SE 17" -- "of" may be removed.